bin/wp-env-config.sh

```diff
@@ -12,11 +12,51 @@ else
   EXIT_CODE=$?
 fi
 
-## set permalinks for easier wp-json
+# PERMALINK SETTINGS ###########################################################
 wp rewrite structure '/%postname%/'
 wp rewrite flush
+
+# PLUGIN SETTINGS ##############################################################
+wp plugin install woo-gutenberg-products-block
```

Comment:
Is this intentional? We have already activated the plugin [here](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/5823/files#diff-9fd21b09a0144355cfd001e3952767df08b2e04241994470aa1d627f797b171eL6).

Reply:
While I probably don't use this PR, `wp plugin install woo-gutenberg-products-block` was intentional. We're want to be able to test the translated cart and checkout block. To accomplish that, we need to fetch the translations from WordPress.org. The problem that we're facing is, that we're using the slug  `woocommerce-gutenberg-products-block` internally, but on WordPress.org, the plugin is using the slug `woo-gutenberg-products-block`. By installing (but not activating) the plugin with the slug `woo-gutenberg-products-block`, I'm able to fetch the corresponding translation files from WordPress.org. 

It's rather a dirty hack, than best practice, though. 🙈

Reply:
Thanks for the clarification! It makes sense to me now!